### DISCUSSION TITLE
Allow site wide setting to default new collections to no access

### DISCUSSION THREAD

**Discussion by routerino** [outline/outline#5727] [Ideas]
An issue when it comes to allowing access to outline for a large internal audience is the pollution of the sidebar by people "playing around".

There is a site wide option for preventing creation of new collections, but that self-serviceability is still a nice value add.

It would be good to be able to force collections to being unshared by default, and forcing users to have to add relevant groups manually.

**Comment by tommoor** [comment]
I think first I would like to redesign the collection creation screen a little and make the access clearer, it's been on the cards for a while.  We can see if that will help with accidental creation of public collections, the default _may_ end up changed as part of that redesign.

If it doesn't help then a config would be the way to go
